src/coreclr/jit/emitarm.cpp

```diff
@@ -6927,8 +6927,18 @@ void emitter::emitDispReg(regNumber reg, emitAttr attr, bool addComma)
 {
     if (isFloatReg(reg))
     {
-        const char* size = attr == EA_8BYTE ? "d" : "s";
-        printf("%s%s", size, emitFloatRegName(reg, attr) + 1);
+        if (attr == EA_8BYTE)
+        {
+            char     regNum[2];
+            unsigned regIndex = reg - REG_F0;
+            regIndex >>= 1;
+
+            printf("d%s", _itoa(regIndex, regNum, 10));
+        }
+        else
+        {
+            printf("s%s", emitFloatRegName(reg, attr) + 1);
```

Comment:
Is this side also incorrect?  I could easily have misread the chain of calls, but it looked like emitFloatRegName is going to return "f<n>" so it would be "sf<n>", which doesn't seem right.

Reply:
Right, but since we do ` + 1`, we remove the extra `f` character that gets printed. This portion is unchanged actually.

Reply:
I will just use something manual version of `_itoa` to convert from int -> string.